CMakeLists.txt

```diff
@@ -394,6 +394,8 @@ SET(ENABLE_NLS 1 CACHE BOOL "Define whether program translation should be suppor
 IF(ENABLE_NLS)
      ADD_DEFINITIONS(-DENABLE_NLS)
 ENDIF()
+# This option toggle manipulations with files in po directory
+SET(ENABLE_PODIR 1 CACHE BOOL "Define whether program translation should be supported.")
```

Comment:
The description should differ from that for `ENABLE_NLS`.

Reply:
Fixed, thanks!